Source/Behaviors/SnapToPointLayoutBehavior.swift

```diff
@@ -95,40 +95,58 @@ public class SnapToPointLayoutBehavior: BrickLayoutBehavior {
         resetCollectionViewContentInset(collectionViewLayout)
     }
 
+    internal func getFilteredAttributes(layout collectionViewLayout: UICollectionViewLayout, frame: CGRect) -> [BrickLayoutAttributes] {
```

Comment:
I don't like the `get` prefix. That's very java-like
Please rename to
`filteredAttributes(layout collectionViewLayout: UICollectionViewLayout, frame: CGRect) -> [BrickLayoutAttributes]`